File: myapp/app/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private EditText num1;
    private EditText num2;
    private Button add;
    private TextView answer;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        num1 = (EditText) findViewById(R.id.editTextNumber);
        num2 = (EditText) findViewById(R.id.editTextNumber2);
        add = (Button) findViewById(R.id.button);
        answer = (TextView) findViewById(R.id.textView2);

        add.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                int number1 = 0;
                int number2 = 0;
                if (!num1.getText().toString().equals("Enter First Number"))
                    number1 = Integer.parseInt(num1.getText().toString());
                if (!num2.getText().toString().equals(null))
                    number2 = Integer.parseInt(num2.getText().toString());
                int sum = number1 + number2;
                answer.setText("The answer is " + String.valueOf(sum));
            }
        });
    }
}

File: app/src/main/java/com/example/tufoodtrucksloginscreen/MainActivity.kt
package com.example.tufoodtrucksloginscreen


import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentPagerAdapter
import androidx.viewpager.widget.ViewPager
import com.alan.alansdk.AlanConfig
import com.alan.alansdk.button.AlanButton
import com.example.tufoodtrucksloginscreen.ui.login.LoginFragment


class MainActivity : AppCompatActivity() {
    private var toTrucks: Button? = null
    private var alanButton: AlanButton? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val viewPager = findViewById<ViewPager>(R.id.viewPager)
        val pagerAdapter = AuthenticationPagerAdapter(
            supportFragmentManager
        )
        pagerAdapter.addFragment(LoginFragment())
        pagerAdapter.addFragment(fragment_register())
        viewPager.adapter = pagerAdapter
        toTrucks = findViewById<Button>(R.id.button6)
        toTrucks!!.setOnClickListener {
            val i = Intent(this, TruckDisplayLoad::class.java)
            startActivity(i)
        }

        val config = AlanConfig.builder().setProjectId("").build()
        alanButton = findViewById(R.id.alan_button)
        alanButton?.initWithConfig(config)
    }


    internal inner class AuthenticationPagerAdapter(fm: FragmentManager?) :
        FragmentPagerAdapter(fm!!) {
        private val fragmentList: ArrayList<Fragment> = ArrayList()
        override fun getItem(i: Int): Fragment {
            return fragmentList[i]
        }

        override fun getCount(): Int {
            return fragmentList.size
        }

        fun addFragment(fragment: Fragment) {
            fragmentList.add(fragment)
        }
    }
}